package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "main": "src/index.js",
   "engines": {
-    "node": "16.x.x"
+    "node": "^16.12.0"
```

Comment:
Actually why do we go for a different version in the ci than for running the project?

Reply:
Because I don't know which version you guys have installed (I'm currently on `16.13.0`), but as long as your greater or equal to this (and less than 17.0.0) it should run on everyone of our machines.

Reply:
I'd vote for keeping the running version the same, and we all switch to the same one. What'cha think?

Reply:
What do you mean by "running version"? The version we run in production? If yes, I find it a good idea that we all switch to it also locally. Nevertheless I have to specify a range here that indicates what node version you have to install to work with this repository.

Reply:
I get that, I was thinking of going with `~16.13.0` and doing a minor lock, cause in the last minor update between 16.11 and 16.12 there was a breaking change basically.

Reply:
There was a breaking change in an experimental feature, yes. I don't know, I've never seen a repo that's locked to a minor node version, usually they're locked to major versions (but we can't do that in this special case, as anything below 16.12.x won't work). For us three it won't make a difference, but as it is a public library, I would find it too restricting.

Reply:
Ok then let's keep it less restrictive then.